src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2306,6 +2363,11 @@
                 }
             },
 
+            "guid": {
```

Comment:
>guid [](start = 13, length = 4)

Ultra-nit: We tend to place identifiers at the top of each object's property list.